crates/rune/src/compile/meta.rs

```diff
@@ -363,6 +387,8 @@ pub(crate) struct ItemMeta {
     pub(crate) visibility: Visibility,
     /// The module associated with the item.
     pub(crate) module: Arc<ModMeta>,
+    /// Doc comment attributes, if any.
+    pub(crate) docs: Arc<Vec<Doc>>,
```

Comment:
`const` items proved to be difficult when doc content was placed in `PrivMeta`. `ItemMeta` let them hold docs without too much friction, and seems to be a much better place for them to live, anyway.

Reply:
Might revisit later exactly how it's done (e.g. maybe we should prefer storing docs indexed separately in `Query`) but it's good enough for now.